util/dvsim/Launcher.py

```diff
@@ -210,8 +215,9 @@ def _get_n_lines(pos, num):
 
             return ''.join(lines[pos:pos + num - 1]).strip()
 
+        err_msg = None
         if self.deploy.dry_run:
-            return True
+            return "P", err_msg
```

Comment:
If I understand this right, `err_msg` is always `None` if we return a status of `"P"` and non-None if we return a status of `"F"`. It might be clearer to return `("P", None)` explicitly here and (to document what's going on) assert `err_msg is not None` before the return on line 258.

Reply:
Done.